uk.co.compucorp.civicrm.tasksassignments/js/src/tasks-assignments/controllers/modal/modal-document.controller.js

```diff
@@ -71,8 +71,10 @@ define([
     vm.viewFile = viewFile;
 
     (function init () {
+      $rootScope.isLoading = false;
       angular.copy(data, vm.document);
       angular.copy(files, vm.files);
+
       vm.data = data;
```

Comment:
please put all this assignments in a separate functions (or multiple functions, depending if the assignments are related or not), this `init` function is too big

Reply:
Done!